pkg/deployments/tasks/20220628-gauge-adder-v2/readme.md

```diff
@@ -1,4 +1,6 @@
-# 2022-06-28 - GaugeAdder
+# 2022-06-28 - GaugeAdder V2
+
+> This Gauge Adder has been deprecated in favor of an [updated version](../../20221111-gauge-adder-v3), which uses the Authorizer Adaptor Entrypoint.
```

Comment:
We should potentially wait until the migration is complete before marking this as deprecated? If we do want to deprecate this now then we should move it to the `deprecated` folder (and update readme).